src/client.js

```diff
@@ -597,29 +598,31 @@ class Client extends EventEmitter {
     // Private functions
     //
     _send(message) {
+        const preparedMessage = { ...message };
         if (!this.connected) {
             return false;
         }
-        message.id = ++this._messageID;
-        message.seq = message.id;
-        this._pending[message.id] = message;
-        this.ws.send(JSON.stringify(message));
-        return message;
+        this._messageID = this._messageID + 1;
+        preparedMessage.id = this._messageID;
+        preparedMessage.seq = preparedMessage.id;
+        this._pending[preparedMessage.id] = message;
+        this.ws.send(JSON.stringify(preparedMessage));
+        return preparedMessage;
     }
 
 
-    _apiCall(method, path, params, callback, callback_params) {
+    _apiCall(method, path, params, callback, callback_params = {}) {
         let isForm = false;
+        let safeMethod = method;
         if (typeof method !== 'string') {
             isForm = true;
-            method = method.method;
+            safeMethod = method.method;
```

Comment:
This was already refactored in my other PR. We should properly destruct this.

Reply:
Yeah, I've tried to do the same thing like in your PR, but eslint warning remains where it was.
As I see your way is more clever assignment, but it doesn't eliminate eslint warning :)

Reply:
I saw no more errors or warnings after running `yarn test`.